### ISSUE TITLE
perf: SWA budget check throttles decode concurrency for hybrid models (MiMo-V2-Flash 16→64 regression)

### ISSUE THREAD

**Issue by pengchengneo** [sgl-project/sglang-jax#1227]
## Summary

PR #921 (`28244725`, 2026-04-17) changed tree_cache initialization order in `scheduler.py`, causing hybrid models like MiMo-V2-Flash to use `SWARadixCache` instead of `ChunkCache` when `disable_radix_cache + chunked_prefill` is set. This leads to `cur_rem_tokens_for_dp()` in `schedule_policy.py` applying `min(full_available, swa_available)` for prefill admission, where SWA budget is depleted at `input_len` rate instead of `sliding_window_size` rate — throttling decode concurrency by ~4x.

## Impact

On 16-chip v6e (dp=4, tp=16, ep=16) with MiMo-V2-Flash:

| | Before #921 (cef4a181) | After #921 (main) |
|---|---:|---:|
| Decode concurrency | **64** | **16** |
| Gen throughput | ~2500 tok/s | ~1050 tok/s |
| Full pool usage | 90% | 20% |
| SWA pool usage | 14% | 4% |

Pool sizes are actually **larger** on main. The bottleneck is purely in admission logic.

## Root Cause

`schedule_policy.py:cur_rem_tokens_for_dp()`:

```python
if self.is_hybrid:
    available = min(
        full_available + full_evictable,
        swa_available + swa_evictable,    # ← bottleneck
    )
```

For MiMo-V2-Flash: `sliding_window_size=128`, `input_len=16384`. Each request deducts 16384 from SWA budget, but SWA layers only need 128 tokens of KV per request. SWA budget is over-estimated by **128x**, blocking admission after 4 requests per DP rank.

## Reproduction

```bash
# Server (16-chip v6e, 4 nodes)
python -m sgl_jax.launch_server \
  --model-path /models/MiMo-V2-Flash --trust-remote-code \
  --tp-size 16 --dp-size 4 --ep-size 16 --moe-backend epmoe \
  --page-size 256 --context-length 262144 --disable-radix-cache \
  --chunked-prefill-size 2048 --dtype bfloat16 --mem-fraction-static 0.95 \
  --swa-full-tokens-ratio 0.2 --max-running-requests 128 \
  --dp-schedule-policy round_robin ...

# Benchmark
python -m sgl_jax.bench_serving --backend sgl-jax \
  --dataset-name random --random-input-len 16384 --random-output-len 1024 \
  --request-rate 100 --num-prompts 256 --max-concurrency 64

# Observe: #running-req stuck at 16, full usage ~20%, swa usage ~4%
# Expected: #running-req ~64-84
```

Works correctly on commit `cef4a181` (branch `epic/mimo-v2-flash`), regresses on `28244725` and later.

## Suggested Fix

In `cur_rem_tokens_for_dp()`, SWA available should not limit admission since SWA layers auto-evict KV beyond `sliding_window_size`:

```python
if self.is_hybrid:
    available = (
        full_available + full_evictable
        # SWA pool evicts beyond sliding_window — don't let it block admission
    )
```

Or scale SWA budget by `sliding_window_size / input_len` before comparing.
